### PR TITLE
Update for AnyConnect already installed

### PR DESCRIPTION
**Pull request by SimonKrughoff**
If you already have AnyConnect installed, you don't need to go through the process of installing it, you can simply add the correct endpoint to the AnyConnect client.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -13,6 +13,8 @@ Currently access is restricted to those IPs on the allowed list.
 At the moment, this is only addresses in the Tucson/AURA network.
 However, others may gain access via the NCSA hosted VPN. Follow instructions in the "Logging into the NCSA VPN" section of `the PDAC wiki <https://confluence.lsstcorp.org/display/DM/PDAC+networking+and+user+accounts+for+developers>`_.
 
+If you happen to already have the AnyConnect client installed, you can skip the above step and just add `vpn.ncsa.illinois.edu` to the AnyConnect connection window.
```
Comment (jonathansick) [inline index.rst]: If you've previously connected to the NCSA VPN from your computer, you can instead launch the AnyConnect application and enter `vpn.ncsa.illinois.edu` in the connection window.

**Review by jonathansick — COMMENTED** [review]
How about the wording, above?

File: index.rst [hunk 2]
```diff
@@ -11,7 +11,9 @@ Authentication
 
 Currently access is restricted to those IPs on the allowed list.
 At the moment, this is only addresses in the Tucson/AURA network.
-However, others may gain access via the NCSA hosted VPN. Follow instructions in the "Logging into the NCSA VPN" section of `the PDAC wiki <https://confluence.lsstcorp.org/display/DM/PDAC+networking+and+user+accounts+for+developers>`_.
+However, others may gain access via the NCSA hosted VPN using the AnyConnect VPN client. Follow instructions in the "Logging into the NCSA VPN" section of `the PDAC wiki <https://confluence.lsstcorp.org/display/DM/PDAC+networking+and+user+accounts+for+developers>`_.
+
+If you've previously previously installed AnyConnect for any reason, you can instead launch the AnyConnect application and enter vpn.ncsa.illinois.edu in the connection window.
```
Comment (SimonKrughoff) [inline index.rst:16]: @jonathansick how about this wording?
